.github/workflows/main.yml

```diff
@@ -135,19 +135,21 @@ jobs:
       - name: Setup emsdk
         shell: bash -l {0}
         run: |
-          emsdk install ${{matrix.emsdk_ver}}
+          cd $HOME
+          git clone https://github.com/emscripten-core/emsdk.git
+          cd emsdk
+          ./emsdk install  ${{ matrix.emsdk_ver }}
```

Comment:
How about we also add version `3.1.58` and/or `latest` to the matrix?

Reply:
Yes this was very beneficial for me atleast for xeus-cpp-lite. I think it would surely help with xeus-r !

Reply:
Wait actually no, I think there is a flaw here. 

We fetch most things like r-base etc from emscripten-forge through environment-wasm-host.yml 

We would want atleast the important packages in the toolchain like r-base, xeus-lite etc to be built against these versions too. (P.S: when I built xeus-cpp-lite against emsdk 3.1.73 ... first I had build errors cause llvm was build against 3.1.45 and once I addressed that, there were build errors from xeus-lite (some symbol mismatch or not exported) and then I built that against 3.1.73 and got xeus-cpp-lite working)

Reply:
So I think we should stick to 1 emsdk version for all pacakges for now (and I guess once emscripten-forge migrates to a new version like latest i.e. 3.1.73 on a separate branch) we can do this.